packages/server/src/sdk/app/applications/sync.ts

```diff
@@ -0,0 +1,53 @@
+import env from "../../../environment"
+import { syncGlobalUsers } from "../../../api/controllers/user"
```

Comment:
The sdk package reaching out into controllers is something we'll want to avoid. I think the right thing to do here is to populate a very minimal `sdk/users` package with the `syncGlobalUsers` added to it